packages/tracing/src/browser/request.ts

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable max-lines */
-import type { Baggage, Span } from '@sentry/types';
+import type { DynamicSamplingContext, Span } from '@sentry/types';
```

Comment:
I'm so glad we have a good test suite for this file (integration tests) - otherwise its tooooo spooky to touch.

Reply:
Agree!